src/check-for-harmful-changes.js

```diff
@@ -0,0 +1,29 @@
+/*
+  getChangeFiles
+  ====
+  - check that the arguments are defined and are Arrays
+  - check if there are potentially harmful changes
+*/
+
+require('@babel/polyfill')
+
+module.exports = (changedFilesArray, ciignoreFilesArray, errorCallback) => {
+  if (typeof ciignoreFilesArray === 'undefined') {
+    console.error(`Harmless Changes: ${ciignoreFilesArray} is undefined.`)
+    return errorCallback()
+  } else if (!Array.isArray(ciignoreFilesArray)) {
+    console.error(`Harmless Changes: ${ciignoreFilesArray} is not an Array.`)
+    return errorCallback()
```

Comment:
You don't need to check for `undefined`. If you pass `undefined` to `isArray`, it will be false and you can log something a little more generic